hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -469,6 +476,7 @@ void innerGetAndApplyDeltaUpdatesFromOM(long fromSequenceNumber,
     LOG.info("Number of updates received from OM : {}, " +
             "SequenceNumber diff: {}, SequenceNumber Lag from OM {}.",
         numUpdates, getCurrentOMDBSequenceNumber() - fromSequenceNumber, lag);
+    return null != dbUpdates ? dbUpdates.isDBUpdateSuccess() : false;
```

Comment:
nit:

```suggestion
    return null != dbUpdates && dbUpdates.isDBUpdateSuccess();
```

Reply:
@adoroszlai - I have handled this suggestion. Pls re-review.